src/ansys_sphinx_theme/__init__.py

```diff
@@ -413,10 +414,24 @@ def convert_pdf_to_png(pdf_path: pathlib.Path, output_dir: pathlib.Path, output_
         images[0].save(output_dir / output_png, "PNG")
     except Exception as e:
         raise RuntimeError(
-            f"Failed to convert PDF to PNG: {e}, ensure `poppler` is installed. See https://pypi.org/project/pdf2image/"  # noqa: E501
+            f"Failed to convert PDF to PNG: {e}. Ensure the PDF file is valid and poppler is installed."  # noqa: E501
         )
 
 
+def run_quarto_command(command, cwd):
+    """Run quartoi command and logs its output."""
+    try:
+        result = subprocess.run(command, cwd=cwd, check=True, capture_output=True, text=True)
+        if result.stdout:
+            logger.info(result.stdout)
+
+        if result.stderr:
+            logger.warning(result.stderr)
+
+    except subprocess.CalledProcessError as e:
+        raise RuntimeError(f"Failed to run the command: {e}")
+
+
```

Comment:
If this is named `run_quarto_command`, then let's ensure it is actually running the tool:

```suggestion
def run_quarto_command(command, cwd):
    """Run quartoi command and logs its output."""
    command = ["quarto"].extend(command)
    try:
        result = subprocess.run(command, cwd=cwd, check=True, capture_output=True, text=True)
        if result.stdout:
            logger.info(result.stdout)

        if result.stderr:
            logger.warning(result.stderr)

    except subprocess.CalledProcessError as e:
        raise RuntimeError(f"Failed to run the command: {e}")


```